src/importexport/musicxml/internal/musicxml/exportxml.cpp

```diff
@@ -660,6 +660,8 @@ static QString color2xml(const EngravingItem* el)
 {
```

Comment:
As color can be seen as belonging to layout, it might make sense to make this depandant of `configuration()->musicxmlExportLayout()`

Reply:
Yes, I was wondering the same. What about line-style?

Reply:
Same, I guess.
For color and export it'd be easy, but for color import it is not

Reply:
But needed in àddColorAttr()` below too